frontend/src/features/moti/mapSideBar/containers/MotiInventoryContainer.tsx

```diff
@@ -178,11 +187,11 @@ export const MotiInventoryContainer: React.FunctionComponent = () => {
           ltsaTitleResponse?.data,
           geocoderResponse,
         );
+        searchRef?.current?.resetForm();
```

Comment:
reset the search fields after the search completes, to prepare the search fields for the next search.